build-setup/linux.md

```diff
@@ -237,6 +237,67 @@ sudo make install
 
 to complete the build.
 
+### CMake
+
+CMake version 3.5 is the minimum required to build PyTorch. Download version 3.19.3 from <https://github.com/Kitware/CMake/releases/download/v3.19.3/cmake-3.19.3-Linux-x86_64.sh> and install:
+
+```
+chmod +x cmake-3.19.3-Linux-x86_64.sh
+mkdir /usr/local/cmake
+./cmake-3.19.3-Linux-x86_64.sh --skip-license --prefix=/usr/local/cmake
+```  
+
+In your `.bashrc` file add CMake to `PATH`:
+
+```
+PATH=$PATH:/usr/local/cmake/bin
+```
```

Comment:
You could include this bit on line 20 so the person working through the instructions can do that all in one go.